File: solutions/two_sum.go
package leetcode

func twoSum(nums []int, target int) []int {
	res := make([]int, 0)
	for i, e := range nums {
		for j := i + 1; j < len(nums); j++ {
			if e+nums[j] == target {
				res = append(res, i, j)
				return res
			}
		}
	}
	return res
}

File: solutions/two_sum.js
var twoSum = function(nums, target) {
    const hashmap = new Map();
    let output;
    
    for (let i = 0; i < nums.length; i++) {
        const num = nums[i];
        const value = hashmap.get(target - num)
        if(hashmap.has(target - num)) {
            output = [value, i]
        } else {
            hashmap.set(num, i);
        }
    }
    
    return output;
};
